.gitignore

```diff
@@ -1,4 +1,6 @@
 *.MOV
 objtrack_env
 .vscode
-yolov3
\ No newline at end of file
+yolov3
+input
```

Comment:
I would rather keep `input/` and `output/` and remove them from `.gitignore` to provide an example along with the code

Reply:
Generally control flow objects (`./input` in this case) are not committed, specially output code/results (`./output` in this case). It also grows the size of the code base to download. GitHub provides a section so you can show a demo of your code.